extension/popup/popup.css

```diff
@@ -119,12 +119,11 @@ body,
 
 #moz-voice-privacy {
   line-height: 20px;
-  margin-right: 32px;
+  margin-left: -16px;
 }
-
 #moz-voice-privacy > a {
   font-weight: 400;
-  font-size: 0.85em;
+  font-size: 0.95em;
```

Comment:
I'm guessing that only this popup.css change is intended to be included here?

There's also changes related to music players.

Reply:
@ianb Yes, I've worked on only popup.css file.

Reply:
@ianb and also, I am not sure why these all other file changes are reflecting in my pull request. I have only made two changes to my popup.css file for this issue.